mask_rcnn/README.md

```diff
@@ -0,0 +1,159 @@
+# Mask R-CNN
+
+## Description
+This model is a real-time neural network for object detection that detects 80 different [classes](https://onnxzoo.blob.core.windows.net/models/opset_10/mask_rcnn/coco_classes.txt).
```

Comment:
We've been categorizing this as an image segmentation model

Reply:
thx, updated